network-api/networkapi/templates/pages/base.html

```diff
@@ -56,7 +56,6 @@
 
         {% block stylesheets %}
             <link rel="stylesheet" href="{% static "_css/main.compiled.css" %}">
-            {% if debug %}<link rel="stylesheet" href="{% static "_css/tailwind.compiled.css" %}">{% endif %}
```

Comment:
I'm unsure why we needed to load an extra `tailwind.compiled.css` for dev env when we are already including Tailwind classes in `main.scss`, `bg-main.scss`, etc. (i.e., through `@tailwind base; @tailwind components;
@tailwind utilities;`)